akka-actor/src/main/scala/akka/routing/Balancing.scala

```diff
@@ -75,6 +75,7 @@ final case class BalancingPool(
     override val supervisorStrategy: SupervisorStrategy = Pool.defaultSupervisorStrategy,
     override val routerDispatcher: String = Dispatchers.DefaultDispatcherId)
     extends Pool {
+  println(s"XXX pool $this with $nrOfInstances")
```

Comment:
Runaway println?

Reply:
Oops, didn't intend to check those in indeed